### DISCUSSION TITLE
Welcome to nimblestudio-demo-assets Discussions!

### DISCUSSION THREAD

**Discussion by nicosang-aws** [aws-samples/nimblestudio-demo-assets#2] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by roxa2019** [comment]
Hi nicosang, 
How do you solve the Maya color management problem?  We use G4dn.8xlarge instace, and the same nimblestudio-demo-assets file, there is an error massage when opening Maya2022，maya failed to use OpenGL normally, which end up wrong color space when rendering. 
Is there a work round for this issue? Thank you for sharing

**Reply by gdismond** [reply]
Greetings @roxa2019 , during early experimentation with Maya2022, I happened upon this link which help resolve color space issues I was having: https://help.autodesk.com/view/MAYAUL/2022/ENU/index.html?guid=GUID-AE1F72ED-FA00-4E81-8027-785E3D6058F7

Hope this helps!

**Reply by roxa2019** [reply]
Thanks for you reply! 
Actually we are not using any external preference file, but only a Maya default one.   When I click on "Enable Color Management", an error says "Failed to finalize the color transform".   The only thing I can do is change Render engine from OpenGL to DirectX 11,  and Maya render view doesn't work, but batch render works fine.  Unfortunatelly we are using XGen so we cannot use DirectX11.
I just wondering how could you render normally while we're using the same Instance?  I guess there should be some restriction on the Graph Card. Did you guys have any custom setting at hardware or software?

**Reply by roxa2019** [reply]
We are using Win Server2019, which OS do you use?

**Reply by gdismond** [reply]
Hi @roxa2019 , we are using mostly Centos on Linux. I did check the Maya site and am finding DirectX solutions only. Have you directly contacted Autodesk to see if they have any suggestions?

**Comment by CelestialVirtuoso** [comment]
Hello, Nicosang! 
First of all, the spanner character models are amazing and very appealing! 
I stumbled upon the spanner project only recently, so not sure if you're gonna see this comment haha. I downloaded Noa rig to test it and I found that she doesn't have eye corner controls, like Ulysse does. Also, when you increase "show face ctrl level" the smaller lip corner controls doesn't seem to be working/they don't affect the mesh when you move them, same with the septum. 
I'm assuming it wasn't wasn't intended and everything was fine during animation process so I was wondering, if it's possible to update the rig file? Those corner controls for eyes and mouth are gonna give more options to get better shapes and poses! 
Thanks for such a great work!
